Example/FinancialConnections Example/FinancialConnections Example/Playground/PlaygroundView.swift

```diff
@@ -176,6 +176,11 @@ struct PlaygroundView: View {
         }
         .navigationTitle("Playground")
         .navigationBarTitleDisplayMode(.inline)
+        .gesture(DragGesture().onChanged(hideKeyboard))
+    }
+
+    private func hideKeyboard(_ value: DragGesture.Value) {
+        UIApplication.shared.sendAction(#selector(UIResponder.resignFirstResponder), to: nil, from: nil, for: nil)
```

Comment:
that's quite clever, and good naming; I guess the `value: DragGesture.Value` is "needed" because of the `onChanged` definition

Reply:
That's right!